### PR TITLE
Remove trailing whitespace Travis CI was complaining about

### PR DESCRIPTION
**Pull request by seth10**

### CODE REVIEW

**Comment by seth10** [comment]
The last error is `ERROR: do not set execute permissions for source files` for `FILE: Documentation/sphinx/kernel-doc`. I tried a `chmod -x` on it through Cygwin, but that must not have made it in the commit. Could someone on a native UNIX platform try?

**Comment by seth10** [comment]
Never mind, got it! Thanks @ethomson ^~^ ([SO](http://stackoverflow.com/a/21694391/5279083))

**Comment by seth10** [comment]
Now the result is `total: 0 errors, 239 warnings, 13806 lines checked`. Although there are 0 errors, it still fails the checks, I guess warnings are more significant to it than I realized.

**Comment by dlech** [comment]
Hmm.. travis is only supposed to be checking the lines you actually changed. I suspect this has to do with  the fact that I changed the default branch name recently. I'll take the whitespace fixes, but `kernel-doc` is supposed to be executable and documentation builds will break if it is not executable.

**Comment by dlech** [comment]
And FWIW, travis only knows pass or fail based on the return code, so warnings and errors are all the same to travis. We don't care about making travis green on this repository, but the automatic checks are there just to point out obvious problems. I missed all of the whitespace problems because I made such a huge commit with the recent documentation changes.

File: evb/evb_input.c [hunk 1]
```diff
@@ -25,25 +25,25 @@
...
+ *
  *     * - Back button press
  *       - ``KEY_BACKSPACE`` (14)
- * 
+ *
  *     * - Joystick button press
  *       - ``KEY_ENTER`` (28)
- * 
+ *
  *     * - Joystic up
```
Comment (dlech) [inline evb/evb_input.c]: Look like we can fix the spelling of joystic here too. And several repeats below.

**Review by dlech — CHANGES_REQUESTED** [review]

**Comment by seth10** [comment]
Okay, thanks for the feedback! When I saw my other PR had a big "All checks have failed," my thought was to fix that. So yes certainly _don't_ merge the `kernel-doc` permission change.

> We don't care about making travis green on this repository

Got it, should I close this?

**Comment by dlech** [comment]
If you can drop the second commit, I will merge the first one. I also noticed a spelling error near the whitespace changes that can be fixed too.

File: evb/evb_input.c (hunk 1 above)
Comment (seth10) [reply evb/evb_input.c]: Good call! Corrected in cd47d22305c19c51625931d216c0bcafcf8021a5

**Review by dlech — APPROVED** [review]

**Comment by dlech** [comment]
Awesome, thanks!

**Pull request merged**